<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>UIToolsEdit</class>
 <widget class="QDialog" name="UIToolsEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tools Editor</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item row="6" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>52</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" >
    <widget class="QToolButton" name="tbHelp" >
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/Icons/Icons/helpkeyword.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>33</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="7" row="0" column="0" >
    <widget class="QListWidget" name="lwTools" />
   </item>
   <item row="8" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="3" >
    <widget class="QGroupBox" name="gbTool" >
     <property name="title" >
      <string>Tool</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>3</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item row="0" column="1" >
       <widget class="QLabel" name="lIcon" >
        <property name="text" >
         <string>Icon</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lCaption" >
        <property name="text" >
         <string>Caption</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QLineEdit" name="lePath" />
      </item>
      <item row="1" column="0" >
       <widget class="QLineEdit" name="leCaption" />
      </item>
      <item row="1" column="1" >
       <widget class="QToolButton" name="tbIcon" >
        <property name="cursor" >
         <cursor>13</cursor>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="lPath" >
        <property name="text" >
         <string>Path</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QPushButton" name="pbNew" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>New</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/Icons/Icons/editadd.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QPushButton" name="pbDelete" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Delete</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/Icons/Icons/editremove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="Line" name="lMiddle" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QPushButton" name="pbUp" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Up</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/Icons/Icons/buttonup.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="QPushButton" name="pbDown" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Down</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/Icons/Icons/buttondown.png</iconset>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2" >
    <widget class="QPushButton" name="pbFinish" >
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Finish</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/Icons/Icons/buttonok.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbFinish</sender>
   <signal>clicked()</signal>
   <receiver>UIToolsEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>358</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel" >
     <x>428</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
